constellation/docker_util.py

```diff
@@ -66,6 +66,17 @@ def remove_volume(name):
     v.remove(name)
 
 
+def container_stop(container, kill, name):
```

Comment:
this has moved here from constellation.py where it used to be a method, so that it can be reused by the `ConstellationService`